README.md

```diff
@@ -13,25 +15,52 @@ The library has been tested with Node version 0.10.32+.
 
 ## Usage
 
-First, you have to configure the library by passing the public key and secret key from `https://dashboard.omise.co/` to `omise` export, for example:
+###Flow
+
+1. Enter a credit card information.
+2. The card information is encrypted via HTTPS and send directly from client using Omise.js, Card.js or Omise-iOS SDK to Omise server.
```

Comment:
2. The card information is sent via HTTPS directly from the client to Omise Servers using Omise.js, Card.js or Omise-iOS SDK,